src/Main.js

```diff
@@ -0,0 +1,189 @@
+import React, { Component } from 'react'
+import ProfilesPage from './ProfilesPage.js'
+import LandingPage from './LandingPage.js'
+import LogInPage from './LogInPage.js'
+import SignUp from './SignUp.js'
+import firebase from './firebase.js'
+
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+
+import * as actions from './actions.js'
+
+const LANDING = 'landing'
+const LOG_IN = 'login'
+const SIGN_UP = 'signup'
+const PROFILES = 'profiles'
+
+class Main extends Component {
+  constructor (props) {
+    super(props)
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.logIn = this.logIn.bind(this)
+    this.setUser = this.setUser.bind(this)
+    this.clearUser = this.clearUser.bind(this)
+    this.handleChange = this.handleChange.bind(this)
```

Comment:
Still hoping you can figure out how to get the linter to deal with defining functions as
```js
logIn = (e) => {
...
}
```
so that you do not have to do the `.bind()` here any longer.

Reply:
fixed